src/Raven.Client/ServerWide/DatabaseRecord.cs

```diff
@@ -307,4 +309,48 @@ public enum DeletionInProgressStatus
         SoftDelete,
         HardDelete
     }
+
+    public class CompressionConfiguration
+    {
+        public string[] Collections;
+        public bool CompressRevisions;
```

Comment:
The default for revisions is that they are all included, because they are rarely touched.
Can you think of a reason why to have this for only specific collections?

Reply:
Actually I was rather thinking about excluding some revisions from compression. For example a user might have heavy subscription on revisions from a particular collection, so faster access time and lower CPU usage can be preferred in that case.

Reply:
I'm fine with that